jib-maven-plugin/src/main/java/com/google/cloud/tools/jib/maven/JibPluginConfiguration.java

```diff
@@ -127,6 +128,20 @@ public PermissionConfiguration() {}
     }
   }
 
+  /** Configuration for {@code platform} parameter. Defaults to amd64/linux. * */
```

Comment:
Therefore,
```suggestion
  /** Configuration for {@code platform} parameter. */
```

Reply:
Makes sense !! Done

Reply:
You can mark a GitHub conversion "resolved" yourself if you believe the issue is cleanly resolved (but only after you pushed new changes).